### PR TITLE
Add login auth for mental health app

### PR DESCRIPTION
**Pull request by ChrisYG**

### CODE REVIEW

**Comment by KentShikama** [comment]
Is `def therapist(request, therapist_id):` covered?

**Comment by KentShikama** [comment]
Also your first commit is named the same as your second and it really doesn't do anything so do you want to go ahead and squash your commits? See https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

File: aspc/mentalhealth/templates/mentalhealth_home.html [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+        </form>
+
+        {% for therapist in therapists %}
+                    <hr/>
+
+            {% include 'therapists/therapist_info.html' with t=therapist %}
+        {% endfor %}
+    {% else %}
+    <p>Pomona students, <a href="{% url 'aspc.auth2.views.login' %}?next={{ absolute_uri|urlencode:"" }}">log in</a> to see the information of therapists.</p>
+    <p>5C students, we want you to be able to use this app too, but your ITS departments won't let us grant you access. <a href="http://itattcc.claremont.edu/project/team/">Submit a complaint on this page</a> if you're angry!</p>
```
Comment (mattdahl) [inline aspc/mentalhealth/templates/mentalhealth_home.html]: Nice to see that this message is still being used :)

**Comment by ChrisYG** [comment]
`def therapist(request, therapist_id):` is covered now and the commits are squashed.

**Pull request merged**